From 57bbfa2e84464dd3a7eb57458b98283f81d2a0ad Mon Sep 17 00:00:00 2001
Message-Id: <57bbfa2e84464dd3a7eb57458b98283f81d2a0ad.1505293157.git.jan.steffens@gmail.com>
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Wed, 13 Sep 2017 07:54:31 +0200
Subject: [PATCH 1/2] Fix bashism in configure.in

Resulted in pkg-config never getting used for libxml if sh is not bash.
---
 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index d6aa3664e6aab08c..f4a0219eecbab9d4 100644
--- a/configure.in
+++ b/configure.in
@@ -505,7 +505,7 @@ dnl
 dnl Try pkg-config first if nothing is set
 dnl
 
-if test "x$LIBXML_CONFIG_PREFIX" == "x" -a "x$LIBXML_SRC" == "x"
+if test "x$LIBXML_CONFIG_PREFIX" = "x" -a "x$LIBXML_SRC" = "x"
 then
 	PKG_CHECK_MODULES([LIBXML], [libxml-2.0 >= $LIBXML_REQUIRED_VERSION], [
 		LIBXML_MANUAL_SEARCH=no
-- 
2.14.1

